src/tests/SimTest.cpp

```diff
@@ -282,17 +282,17 @@ void SimTest::updateObjectLightSetupRGBAObservation() {
   CORRADE_VERIFY(objectID != esp::ID_UNDEFINED);
   simulator->setTranslation({1.0f, 0.5f, -0.5f}, objectID);
   checkPinholeCameraRGBAObservation(
-      *simulator, "SimTestExpectedDefaultLighting.png", maxThreshold, 0.65f);
+      *simulator, "SimTestExpectedDefaultLighting.png", maxThreshold, 0.71f);
 
   // change from default lighting to custom
   simulator->setObjectLightSetup(objectID, "custom_lighting_1");
   checkPinholeCameraRGBAObservation(
-      *simulator, "SimTestExpectedCustomLighting.png", maxThreshold, 0.65f);
+      *simulator, "SimTestExpectedCustomLighting.png", maxThreshold, 0.71f);
```

Comment:
Um, these don't necessarily need to be same everywhere ;) For some scenes the difference might be higher, for some not, and making the thresholds unnecessarily large might hide errors.

Reply:
My tests were failing on all of these scenes with the same error ~0.702.

Reply:
oh, interesting -- in my case it was just one of these.

Fair enough, then :)